databricks/koalas/series.py

```diff
@@ -3158,34 +3158,47 @@ def idxmax(self, skipna=True):
 
         >>> s.idxmax()
         ('b', 'f')
+
+        If multiple values equal the maximum, the first row label with that
+        value is returned.
+
+        >>> s = ks.Series([1, 100, 1, 100, 1, 100], index=[10, 3, 5, 2, 1, 8])
+        >>> s
+        10      1
+        3     100
+        5       1
+        2     100
+        1       1
+        8     100
+        Name: 0, dtype: int64
+
+        >>> s.idxmax()
+        3
         """
+        if len(self) == 0:
+            raise ValueError("attempt to get idxmin of an empty sequence")
```

Comment:
Why moved here?

Reply:
Ah, at the first time i thought it can reduce the cost since `if len(self) == 0` then we can skip all process after this line,

but now i'm thinking maybe i missed case that if the size of `self` is extremely big, it can rather reduce the performance.

i'll move this logic to more proper place.